### PR TITLE
Add support and licences filter from old Ralph

### PR DESCRIPTION
**Pull request by szok**
- Added Support filters
- Added Licence filters
- Added new Decimal filter
- Rename Support SoftwareCategory model to Software
- Added asset_type to Support Software model

### CODE REVIEW

File: src/ralph/admin/filters.py [hunk 1]
```diff
@@ -275,6 +305,7 @@ def register_custom_filters():
     """
     field_filter_mapper = [
         (lambda f: bool(f.choices), ChoicesListFilter),
+        (lambda f: isinstance(f, models.DecimalField), DecimalListFilter),
```
Comment (mkurek) [inline src/ralph/admin/filters.py]: what about integer field?

File: src/ralph/admin/filters.py [hunk 2]
```diff
@@ -187,6 +187,36 @@ def choices(self, cl):
...
+                    '{}__gte'.format(self.field_path): value[0]
+                })
+            if value[1]:
+                queryset = queryset.filter(**{
+                    '{}__lte'.format(self.field_path): value[1]
+                })
+
+            return queryset
+
+    def choices(self, cl):
```
Comment (mkurek) [inline src/ralph/admin/filters.py:209]: do we have to use choices here (and in date filter too)? I don't like this yielding-only-single-element.

File: src/ralph/admin/templates/admin/filters/decimal_filter.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<h6>
+  {% blocktrans with filter_title=title %}
+    {{ filter_title }}
+  {% endblocktrans %}
+</h6>
+{% with choices.0 as i %}
+  <input type="text" name="{{ i.parameter_name_start }}" placeholder="From" value="{{ i.start_value }}" />
```
Comment (mkurek) [inline src/ralph/admin/templates/admin/filters/decimal_filter.html]: what do you think about `<input type="number" step="any" ...>`?

File: src/ralph/licences/models.py [hunk 4]
```diff
@@ -73,13 +74,14 @@ class Licence(
...
     )
-    # property_of = models.ForeignKey(
-    #     settings.AUTH_USER_MODEL,
-    #     on_delete=models.PROTECT,
-    #     null=True,
-    # )
-    software_category = models.ForeignKey(
-        SoftwareCategory,
+    property_of = models.ForeignKey(
+        'back_office.AssetHolder',
```
Comment (mkurek) [inline src/ralph/licences/models.py]: wouldn't it be better to define `AssetHolder` in assets app? it'll be used in BO, DC, Domains and maybe somewhere else in the future

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**